### PR TITLE
526 deploy application

### PR DESCRIPTION
**Pull request by johnake**
deploy application

Include relevant motivation and context.

### Changes proposed in this pull request

Deployment of the application into AKS in the development environment

### Guidance to review

Check the application is created in AKS - pod running with no errors in the logs
Check the site is up and running: https://find-a-lost-trn-development.test.teacherservices.cloud/

### Checklist

- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by johnake** [comment]
[Deploy application](https://trello.com/c/6MLU57cY/526-deploy-application)

File: Makefile [hunk 1]
```diff
@@ -192,6 +192,9 @@ terraform-init-aks: bin/terrafile
 	./bin/terrafile -p terraform/aks/vendor/modules -f terraform/aks/workspace_variables/$(CONFIG)_Terrafile
 	terraform -chdir=terraform/aks init -backend-config workspace_variables/$(CONFIG).backend.tfvars $(backend_config) -upgrade -reconfigure
```
Comment (nickdevdfe) [inline Makefile:194]: Remove the blank line

File: terraform/aks/application.tf [hunk 2]
```diff
@@ -1,4 +1,43 @@
 locals {
   environment  = "${var.app_environment}${var.app_suffix}"
   service_name = "find-a-lost-trn"
+  app_secrets = {
+    DATABASE_URL = var.deploy_postgres ? module.postgres.url : "${data.azurerm_key_vault_secret.db_url[0].value}"
+    REDIS_URL    = var.deploy_redis ?  module.redis[0].url : "${data.azurerm_key_vault_secret.redis_url[0].value}"
+  }
+}
+
+module "api_application" {
```
Comment (nickdevdfe) [inline terraform/aks/application.tf:10]: Suggest you use web_application

File: terraform/aks/variables.tf [hunk 3]
```diff
@@ -2,6 +2,8 @@ variable "app_environment" {
   type = string
 }
 
+variable "file_environment" {}
```
Comment (nickdevdfe) [inline terraform/aks/variables.tf:5]: define a type, default?, and description.

File: terraform/aks/variables.tf
Comment (nickdevdfe) [inline terraform/aks/variables.tf:1]: Add descriptions, default and types as needed.

**Review by nickdevdfe — APPROVED** [review]
Tested, reviewed and approved.

**Pull request merged**